### PR TITLE
Update team CodeBender's team page for mid of week 2

### PR DESCRIPTION
**Pull request by thngkaiyuan**

### CODE REVIEW

File: teams/codebender.md [hunk 1]
```diff
@@ -19,14 +19,17 @@ permalink: /teams/codebender/
...
+* [x] Project set up - Week 2, Tue
 * [ ] Feature planned
 
 ## Contributions
  
 #### @thngkaiyuan (Thng Kai Yuan)
-* ...
+### Technology expertise in Information Security
+ * Sharing on Cross-Site Request Forgeries ([Blog Post on CSRF](http://cs3281-2.blogspot.sg/2016/01/cross-site-request-forgeries-csrf.html))
+ * Write-up on my participation in Insomni'hack 2016: ([Blog Post on the Smartcat Challenge](http://cs3281-2.blogspot.sg/2016/01/te-insomnihack-2016-teaser-smartcat-web.html))
```
Comment (damithc) [inline teams/codebender.md:32]: Your involvement in NUS Greyhats qualify too.

Comment (thngkaiyuan) [inline teams/codebender.md:32]: That's great! Do I simply add a one-liner "My participation in NUS GreyHats" or would it be better for me to list the activities that I do in there (e.g. Meetup with Cyber Test Systems to discuss cyber range scenarios, Weekly Training: [1], [2], [3])?

Comment (damithc) [inline teams/codebender.md:32]: May be a summary? e.g. attended 3 training sessions, gave a talk on XYZ, 
If in doubt, you can err on the side of too much details. It's easy for us to tell you when there's too much details. Much harder to say anything when details are missing altogether. :-)

Comment (thngkaiyuan) [inline teams/codebender.md:32]: Got it, thanks Dr Damith!

**Pull request merged**